login-workflow/src/new-architecture/components/ChangePasswordDialog/types.ts

```diff
@@ -1,14 +1,19 @@
-import { DialogProps } from '@mui/material';
+import { SxProps } from '@mui/material';
+import { BasicDialogProps } from '../Dialog';
 import { SetPasswordProps } from '../SetPassword';
 
-// we should define a reusable type for the password input props (this should be defined with the reusable password component and reused by CreatePassword, ResetPassword, and ChangePassword).
-export type ChangePasswordDialogProps = DialogProps & { PasswordProps: SetPasswordProps } & {
+export type ChangePasswordDialogProps = { passwordProps?: SetPasswordProps } & {
```

Comment:
`passwordProps` should be `PasswordProps`. When we are defining a prop which will be available for any other child component, we are defining a camel case. Please `PasswordProps` in other components.
Same comment for errorDialogProps